README

```diff
@@ -1,3 +1,30 @@
+*** Description ***
```

Comment:
I think we should have our website URL (ex: "www.wolfssl.com") at the top of the normal README in some form as well.